model/reactions.tsv

```diff
@@ -4045,25 +4043,18 @@ rxns	rxnKEGGID	rxnBiGGID	rxnEHMNID	rxnHepatoNET1ID	rxnREACTOMEID	rxnRecon3DID	rx
 "MAR01624"	"R08733"	"VLCSr"	"R04580R"	"r0703"	""	"VLCSr"	"MNXR105207;MNXR108978"	"HMR_1624"	"RCR13605"	""	0	""	""	"HMR_1624"
 "MAR01625"	"R08733"	"VLCSp"	""	""	""	"VLCSp"	"MNXR105207"	"HMR_1625"	"RCR10914"	""	0	""	""	"HMR_1625"
 "MAR01627"	"R08733"	""	""	""	"R-HSA-193401"	"HMR_1627"	"MNXR112239"	"HMR_1627"	"RCR10394"	""	0	""	""	"HMR_1627"
-"MAR01629"	""	""	""	""	"R-HSA-192325"	"HMR_1629"	"MNXR125295"	"HMR_1629"	"RCR20233"	""	0	""	""	"HMR_1629"
-"MAR01630"	""	""	""	""	"R-HSA-192325"	"HMR_1630"	"MNXR125295"	"HMR_1630"	"RCR20511"	""	0	""	""	"HMR_1630"
+"MAR01629"	""	""	""	""	"R-HSA-192325"	"HMR_1629;HMR_1630"	"MNXR125295"	"HMR_1629;HMR_1630"	"RCR20233;RCR20511"	""	0	""	""	"HMR_1629;HMR_1630;MAR01630"
 "MAR01631"	""	""	""	""	"R-HSA-192325"	"HMR_1631"	"MNXR125295"	"HMR_1631"	"RCR20512"	""	0	""	""	"HMR_1631"
-"MAR01632"	"R08734"	""	"RE2624X"	""	"R-HSA-192056"	"AMACRp;RE2624X"	"MNXR112240"	"HMR_1632"	"RCR10915"	""	0	"RHEA:40456"	"RHEA:40455"	"HMR_1632"
-"MAR01635"	"R08734"	""	"RE2624M"	""	"R-HSA-192056"	"HMR_1635;RE2624M"	"MNXR112240"	"HMR_1635"	"RCR13606"	""	0	"RHEA:40456"	"RHEA:40455"	"HMR_1635"
-"MAR01637"	""	""	"RE3247M"	"r0706"	"R-HSA-192335"	"HMR_1637;RE3247M"	"MNXR105369;MNXR124225"	"HMR_1637"	"RCR13607"	""	0	""	""	"HMR_1637"
-"MAR01638"	""	"ACOX2x"	""	""	""	"ACOX2x"	""	"HMR_1638"	"RCR10916"	""	0	""	""	"HMR_1638"
-"MAR01639"	""	""	"RE3247X"	"r0706"	"R-HSA-192335"	"RE3247X;r0706"	"MNXR105369;MNXR124225"	"HMR_1639"	"RCR10917"	""	0	""	""	"HMR_1639"
-"MAR01642"	"R04813"	""	""	"r0744"	"R-HSA-192331"	"r0744"	"MNXR105377;MNXR109137;MNXR115598"	"HMR_1642"	"RCR10918"	""	0	""	"RHEA:18933"	"HMR_1642"
-"MAR01646"	"R04812"	""	""	"r0743"	""	"r0743"	"MNXR105376;MNXR109136"	"HMR_1646"	"RCR13608"	""	0	""	""	"HMR_1646"
-"MAR01652"	"R03719"	"SCPx"	"R03719X"	"r0628"	""	"SCPx"	"MNXR104297;MNXR108369"	"HMR_1652"	"RCR13609"	""	0	""	"RHEA:16865"	"HMR_1652"
-"MAR01653"	"R03719"	"SCPx"	"R03719X"	"r0628"	""	"HMR_1653"	"MNXR104297;MNXR108369"	"HMR_1653"	"RCR13610"	""	0	""	"RHEA:16865"	"HMR_1653"
-"MAR01656"	""	""	""	"r0990"	""	"r0990"	"MNXR105412"	"HMR_1656"	"RCR20134"	""	0	""	""	"HMR_1656"
-"MAR01659"	""	""	""	"r2501"	""	"r2501"	"MNXR106313"	"HMR_1659"	"RCR20234"	"3.A.1.203.3"	0	""	""	"HMR_1659"
+"MAR01632"	"R08734"	""	"RE2624X"	""	"R-HSA-192056"	"AMACRp;RE2624X;HMR_1635;RE2624M"	"MNXR112240"	"HMR_1632;HMR_1635"	"RCR10915;RCR13606"	""	0	"RHEA:40456"	"RHEA:40455"	"HMR_1632;HMR_1635;MAR01635"
```

Comment:
@haowang-bioinfo Is the last column correct? Shouldn't `rxnRetired` refer to reaction identifiers for the exact same reaction? Because `rxnRetired` here also includes MARxxxxx identifiers of _similar_ reactions that were deleted. For instance, `MAR01632` is the peroxisome variant that is kept, while `MAR01635`. In the current file `MAR01635` is added as `rxnRetired` for `MAR01632`, but I don't think that's correct?

Reply:
> Shouldn't rxnRetired refer to reaction identifiers for the exact same reaction?

good question, and yes the ones in `rxnRetired` should be the same or similar ones to the master id

> In the current file MAR01635 is added as rxnRetired for MAR01632, but I don't think that's correct?

agree, adding `MAR01635` as `rxnRetired` for `MAR01632` is incorrect. Because they are not duplicate to each other

Reply:
ah okay I was thinking of `MAR01635` as the (wrongly) mitochondrial "duplicate" of `MAR01632`; I can undo that
I also added the IDs from other models that were associated with `MAR01635` to the lists of IDs associated with `MAR01632`; should I also undo that? I figured that might be helpful if you were wondering why the mitochondrial version of the reaction in, say, Recon3D doesn't appear to be in Human-GEM, and seeing it in the row for `MAR01632` might convey that it was dropped because the peroxisomal version was the only "correct" one? No idea if that would actually be clear to a hypothetical future observer though, and could just be extra confusing for no reason

Reply:
> No idea if that would actually be clear to a hypothetical future observer though, and could just be extra confusing for no reason

everything should be clear to a future observer with the provided messages in this PR